README.md

```diff
@@ -8,220 +7,299 @@ kerl [![GitHub Actions CI][ci-img]][ci] [![GitHub Actions Lint][lint-img]][lint]
 
 Easy building and installing of [Erlang/OTP](https://www.erlang.org) instances.
 
-Kerl aims to be shell agnostic and its only dependencies, excluding what's
+`kerl` aims to be shell agnostic and its only dependencies, excluding what's
 required to actually build Erlang/OTP, are `curl` and `git`.
 
 All is done so that, once a specific release has been built, creating a new
 installation is as fast as possible.
 
-OTP Support Policy
-------------------
+**Note**: `kerl` also tries to report on required dependencies but 1. it's not guaranteed
+this list is complete, 2. it's not guaranteed that all the elements in the list
+are required (it's on a best-effort basis).
 
-As of 2021 September 17, we are supporting the current OTP release version
-and 2 prior release versions (same as upstream OTP.) Older OTP releases
-may or may not work. We will advance release support as new releases of OTP
-become available.
-
-Triage cadence
---------------
-
-We triage kerl pull requests and issues at least once a month, typically on
-the fourth Tuesday of the month at 1 pm US/Pacific or 8 pm UTC.
-
-IRC channel
------------
+## Erlang/OTP support policy
 
-We have a channel on [Libera](https://web.libera.chat/gamja/) called `#kerl` -
-feel free to join and ask support or implementation questions any time. If
-no one is around, feel free to open an issue with your question or problem
-instead.
```

Comment:
I remove this since I don't believe it's true anymore, but lemme know...
We could also create a Slack channel (don't know if you use it) and advertise it in the Erlang and EEF Slacks. 👍